plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx

```diff
@@ -75,67 +68,53 @@ const useStyles = makeStyles((theme: BackstageTheme) =>
     bold: {
       fontWeight: theme.typography.fontWeightBold,
     },
-    service: {
+    box1: {
       background: createPageTheme(theme, 'home', 'service'),
```

Comment:
I think you might be able to use the actual types here; the declaration can be a function that accepts props:

```suggestion
      background: props => createPageTheme(theme, props.types.length > 1 ? props.types[1] : 'none'),
```

Then call useStyles with the list of types further down:
```
const classes = useStyles({ types: topN.map(c => c.type) });
```

Reply:
I still dont get what you mean, do we really need to solve this now ?

Reply:
What if the entity type is not described in the `pageTheme`, how should I handle that ?  Currently backstage  have 9 themes ( 'home' , 'service', etc...) If it is not match with the available theme list, does assigning a default value a good solution ?

Reply:
updated